config/locales/en.yml

```diff
@@ -2344,9 +2344,17 @@ en:
       legal_1_1_aup_url: https://wiki.openstreetmap.org/wiki/Acceptable_Use_Policy
       legal_1_1_privacy_policy: Privacy Policy
       legal_1_1_privacy_policy_url: https://osmfoundation.org/wiki/Privacy_Policy
+      legal_1_2_html: |
+        If you would like to use OpenStreetMap maps or data please make sure to attribute OpenStreetMap—refer to %{attribution_guidelines_link} for more information.
+      legal_attribution_guidelines_title: Attribution Guidelines
+      legal_attribution_guidelines_url: https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines
+      legal_1_3_html: |
+        Specifically for using a screenshot from OpenStreetMap directly in your book, publication, film or TV show, you do not need to ask for permission as OpenStreetMap is open source. Refer to the %{attribution_guidelines_link} as well as the examples at %{attribution_example_1_link} and %{attribution_example_2_link}
```

Comment:
I'm not sure "open source" is the best phrase? or do we use that elsewhere?

Reply:
I think open data is fine, or maybe something like "has an open license" - will defer to you all

Reply:
I have updated it to "open for all to use". There's the valid concern that people may stop reading there and just use without attribution, but I'm hoping that it being followed immediately by "Refer to the Attribution Guidelines..." should be enough to avoid this.